### PR TITLE
feat/be: integrate train model api [decenter-compute]

### LINKED ISSUE #25: US: Train/NewModel
- [x] New Model
- [x] Use template: is like the demo version 
- [x] Fine Tune: disable it for now

### PR DESCRIPTION
**Pull request by techyNonso**
Resolves #25

### CODE REVIEW

File: src/app/dashboard/train/components/NewModel.tsx [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+    const handleTrainScriptDragOver = (e: React.DragEvent<Blob>) => {
+        e.preventDefault()
+
+    }
+
+    const handleTrainScriptDrop = (e: React.DragEvent<Blob>) => {
+        e.preventDefault()
+        const files = e.dataTransfer.files
+        if (!files || files.length > 1) {
+            console.log("upload one file")
```
Comment (Nasfame) [inline src/app/dashboard/train/components/NewModel.tsx:78]: can i flash the error instead in the ui?

Comment (techyNonso) [reply src/app/dashboard/train/components/NewModel.tsx:78]: What error, conflict error?

File: src/app/dashboard/train/components/NewModel.tsx [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+            return
+        }
+
+        if (files[0].name.match(regex)?.[0] === "ipynb") {
```
Comment (Nasfame) [inline src/app/dashboard/train/components/NewModel.tsx:82]: can u list the train scripts and let files choose.

Comment (techyNonso) [reply src/app/dashboard/train/components/NewModel.tsx:82]: I worked based on the design given

File: src/app/dashboard/train/components/NewModel.tsx [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+            console.log("upload one file")
+            return
+        }
+        console.log(files)
+
+        if (files[0].name.toUpperCase() === "REQUIREMENTS.TXT") {
+            //store file
+            setRequirementsScript(files[0])
+        } else {
+            console.log("upload a requirements.txt file")
```
Comment (Nasfame) [inline src/app/dashboard/train/components/NewModel.tsx:111]: no necessary, lets forget about requirements.txt for now. Please comment or disable for now

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: src/app/dashboard/train/components/NewModel.tsx (hunk 1 above)
Comment (Nasfame) [reply src/app/dashboard/train/components/NewModel.tsx:78]: okay!!

**Comment by Nasfame** [comment]
Fix #25

**Review by Nasfame — APPROVED** [review]
LGTM

File: next.config.js
Comment (Nasfame) [inline next.config.js:1]: @techyNonso take a look on this file

**Review by Nasfame — APPROVED** [review]
LGTM

**Review by Nasfame — APPROVED** [review]
LGTM

**Review by Nasfame — APPROVED** [review]
Did the Merge, Waiting for successfull deployment!!

**Pull request merged**